pkg/controller/disruption/disruption.go

```diff
@@ -173,94 +175,119 @@ func (dc *DisruptionController) finders() []podControllerFinder {
 		dc.getPodStatefulSets}
 }
 
+var (
+	controllerKindRS  = v1beta1.SchemeGroupVersion.WithKind("ReplicaSet")
+	controllerKindSS  = apps.SchemeGroupVersion.WithKind("StatefulSet")
+	controllerKindRC  = v1.SchemeGroupVersion.WithKind("ReplicationController")
+	controllerKindDep = v1beta1.SchemeGroupVersion.WithKind("Deployment")
```

Comment:
We also have Deployments in the apps group. I think you need to use both but I am not fairly certain.